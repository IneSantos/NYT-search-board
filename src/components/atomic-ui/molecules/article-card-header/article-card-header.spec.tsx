import React from "react";
import { render } from '@testing-library/react';
import ArticleCardHeader from "./article-card-header";

describe("ArticleCardHeader component tests", () => {
  test(`ArticleCardHeader component render`, () => {
    const { container } = render(
      <ArticleCardHeader headline={"This is an headline"}/>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
