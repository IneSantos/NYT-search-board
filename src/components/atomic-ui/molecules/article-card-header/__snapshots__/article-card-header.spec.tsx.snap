// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleCardHeader component tests ArticleCardHeader component render 1`] = `
<div
  class="card-header"
>
  <h3
    class="title-small"
  >
    This is an headline
  </h3>
</div>
`;
